databricks/koalas/utils.py

```diff
@@ -113,10 +119,15 @@ def combine_frames(this, *args, how="full"):
                 )
                 for label in that._internal.column_labels
             ]
+            + order_column
         )
 
         index_columns = set(this._internal.index_spark_column_names)
-        new_data_columns = [c for c in joined_df.columns if c not in index_columns]
+        new_data_columns = [
+            c
```

Comment:
nit, I'm just curious, does `c` is generally used for indicating the `column` ?
I think `column` is more clearer than `c` if there is no special reason to should use `c`.
What do you think about? 🤔

Reply:
I just followed the previous commit, but `col` or `column` might be better. I'll update it.

Reply:
Oh, I see. Thanks :)